java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/build/preprocessor/AbstractPreprocessor.java

```diff
@@ -189,7 +189,7 @@ private String getContainerFile() {
             ENV PROXY_URL=$PROXY_URL
             COPY .jbs/run-build.sh /var/workdir
             COPY . /var/workdir/workspace/source/
-            RUN /var/workdir/run-build.sh
+            RUN /var/workdir/run-build.sh | tee /var/workdir/build.log
```

Comment:
What about error output ?

Reply:
I guess mostly what we need in the log file is the mvn output. Not sure if we need to care about stderr.

Reply:
I'm not sure if we can guarantee that Ant/SBT/Maven/Gradle do **not** send any parts of any logs to stderr ? i.e. I'm just trying to make sure we don't miss anything.

Reply:
I google and it seem tee can easily do it, like './xx.sh 2>&1 | tee log' . I will update it.

Reply:
updated.